packages/@sparrow-common/src/components/header/sub-component/PopupHint.svelte

```diff
@@ -74,4 +77,20 @@
     font-size: 12px;
     background-color: var(--bg-tertiary-400);
   }
+  .tooltip-dropdown-data-web {
+    font-weight: 400;
+    z-index: 1;
+    gap: 6px;
+    top: 40px;
+    border-radius: 3px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    height: 25px;
+    width: 335px;
+    font-size: 12px;
+    background-color: var(--bg-tertiary-400);
+    left: -20px;
+  }
```

Comment:
you can handle this dynamically, no need to add check for desktop and web, currently it is hardcoded.

Reply:
I will fix it